code/modules/client/preferences_savefile.dm

```diff
@@ -375,7 +375,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	facial_hair_color			= sanitize_hexcolor(facial_hair_color, 3, 0)
 	eye_color		= sanitize_hexcolor(eye_color, 3, 0)
 	skin_tone		= sanitize_inlist(skin_tone, skin_tones)
-	backbag			= sanitize_integer(backbag, 1, backbaglist.len, initial(backbag))
+	backbag			= sanitize_inlist(backbag, backbaglist)
```

Comment:
this doesn't convert people's savefiles, it just randomises the backbag they have.
Define a conversion like I did for antag roles. (Heck, you only have to sort satchels and normal backpacks (since everything else wasn't available yet))